### DISCUSSION TITLE
Osiset laravel Shopify vs shopifys CLI php

### DISCUSSION THREAD

**Discussion by mikemati** [osiset/laravel-shopify#1211] [General]
Hi I have probably what is an easy question but I can't understand the use cases. 

Shopify now provides creating php apps through the CLI. I know osisets laravel-Shopify has been around for a while. Is there still a use case for it?

**Comment by Kyon147** [comment]
Hi @mikemati 

I've only had a quick look at the PHP CLI offering but here's my thoughts. 

You are free to use the Shopify CLI / PHP install as it does do similar things to this package but there are some differences. 

The Shopify CLI package scaffolds with React, so if you want to use VueJS you might need to spend some time to detangle the two to then install Vue. It also uses React for AppBridge so you'd have to set that back up again.

The other thing, the CLI builds a bare bones package that basically handles OAuth and the React Routing, plus some basic Webhook stuff. There's no built in plans or billing that I saw (albeit a quick look).

We've recently brought in features around theme-extension support which I believe you'd have to implement yourself on the CLI, same with things like script tags.

There's also a lack of documentation with the only bit being here https://github.com/Shopify/shopify-app-template-php/tree/cli_three so you'd have to already have a deep knowledge of how to build apps on Shopify and Laravel to build out the toolsets you need. 

At the end of the day, it will be down to personal preference - so it's up to you which one you use :)

**Reply by mikemati** [reply]
Thanks I appreciate your insight. I agree the documentation is definitely lacking on Shopify's end
